usagov/vote-gov-drupal
# VOTE-421 implement color palette, global link and button styles

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-421

## Description

Implement color palette, link and button styles

## Deployment and testing

### Post-deploy

1. cd into votegov theme and run `npm run build`

### QA/Test

1. log into the site and visit the http://vote-gov.lndo.site/promoting-access-to-voting check that colors are applied
2. visit the http://vote-gov.lndo.site/about-us and check that colors are being applied
3. color variables live in `votegov/src/sass/_variables.scss` file

## Checklist for the Developer




## Checklist for the Peer Reviewers
